virttest/guest_agent.py

```diff
@@ -209,7 +209,7 @@ def _read_objects(self, timeout=READ_OBJECTS_TIMEOUT):
         """
         if not self._data_available():
             return []
-        s = ""
+        s = b""
```

Comment:
There might be some issues at line 231 and 234.

Reply:
updated, thanks